core/variant/variant_utility.cpp

```diff
@@ -1228,6 +1143,15 @@ bool VariantUtilityFunctions::is_same(const Variant &p_a, const Variant &p_b) {
 	return p_a.identity_compare(p_b);
 }
 
+String VariantUtilityFunctions::join(const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
```

Comment:
```suggestion
String VariantUtilityFunctions::join(const Variant **p_args, int p_arg_count) {
```
No need to add an unused argument